addons/l10n_it_edi/models/res_partner.py

```diff
@@ -14,7 +14,7 @@ class ResPartner(models.Model):
 
     l10n_it_pec_email = fields.Char(string="PEC e-mail")
     l10n_it_codice_fiscale = fields.Char(string="Codice Fiscale", size=16)
-    l10n_it_pa_index = fields.Char(string="PA index",
+    l10n_it_pa_index = fields.Char(string="Destination Code",
```

Comment:
I think you need to change the translation too,
and there is another usage of the expression 'PA index' in the constraint just below:

https://github.com/odoo/odoo/blob/3615bf69bf07e681556c88fc5a26e44a019b441a/addons/l10n_it_edi/models/res_partner.py#L31